tests/golden/test_runner.py

```
@@ -1452,5 +1458,99 @@ def test_run_l3_resident_rejects_non_l3(self):
+        self.effective_us = effective_us
+
+
+class _FakeStats:
+    """Stand-in for ``BenchmarkStats`` — only what the reporters read.
+
+    Keeps the unit tests off the installed runtime (the CPU-only job runs with
+    conftest's stub ``pypto``).
+    """
+
+    rounds, warmup, fallback_flattened, all_zero_device = 2, 1, False, False
+    host_wall_us = [300.0, 310.0]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)